tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -112,6 +113,17 @@ def _forward_backward_test_impl(
                 deallocate_pipeline_output=deallocate_pipeline_outputs,
             )
 
+            h_size = PipelineParallelForwardBackwardTest.HIDDEN_SIZE
+            m_size = PipelineParallelForwardBackwardTest.MICRO_BATCH_SIZE
+            target_loss = 0
+            for r in range(self.world_size):
+                target_loss = target_loss + h_size*(r + 1) + 1
```

Comment:
I'm not sure about the arithmetic here, should there be a multiply in the loop if output of the previous stage is being multiplied with the current layer's weights? Alternatively the hidden layer weights could be set to zero.